VMS/Pages/Homepage/SloganAndVideo.razor.scss

```diff
@@ -1,15 +1,191 @@
 ﻿@import '../../Resources/scss/common/_variables';
+@import '../../Resources/scss/common/_animation.scss';
 
-.my-slogan {
-    background-color: $green;
+//---------------MOBILE UI------------------//
+//-----------------------------------------//
+.video-header {
+    position: relative;
+    width: auto;
+    height: auto;
+    margin-top: 85px;
 
-    p {
+    &__note {
+        position: absolute;
+        display: inline-block;
+        width: 100%;
+        height: 41px;
+        top: 0;
+        left: 0;
+        right: 0;
         color: $white;
-        font-size: 20px;
-        font-weight: 700;
+        background-color: $green;
+        opacity: 0;
+        animation: Hidden 6s ease 0.5s 1 backwards;
+
+        &__text {
+            position: relative;
+            display: inline-block;
+            width: 320px;
+            height: 30px;
+            top: 50%;
+            left: 50%;
+            transform: translate(-50%,-50%);
+            font-style: normal;
+            font-weight: 600;
+            font-size: 12px;
+            line-height: 15px;
+            text-align: center;
+        }
+    }
+
+    &__source {
+        position: relative;
+        width: 100%;
+        height: auto;
+        // keep original ratio of the video (fullHD)
+        top: 0;
+        object-fit: cover;
+        background-repeat: no-repeat;
+        z-index: -1;
+    }
+
+    &__title {
+        position: absolute;
+        display: inline-block;
+        text-align: center;
+        left: 50%;
+        bottom: 10%;
+        transform: translate(-50%,0);
+        font-size: 18px;
+        color: $white;
+        font-style: normal;
+        animation: Swipe 1s ease 0.5s 1 backwards;
+
+        &--main {
+            font-weight: bold;
+            font-weight: 700;
+        }
+
+        &--sub {
+            font-weight: normal;
+            font-weight: 500;
+        }
+    }
+}
+
+//---------------BREAKPOINT 576PX------------------//
+//------------------------------------------------//
+@include on('sm') {
+    .video-header {
+        margin-top: 110px;
```

Comment:
screen under 992px, height of navbar is 85px so video must margin-top 85px

Reply:
i solved it.